parser/context.go

```diff
@@ -278,6 +278,7 @@ func specialChildren(node ast.Node) []ast.Node {
 	panic(fmt.Sprintf("specialChildren: Unknown node %#v", node))
 }
 
+// Children returns the direct, thunk and special children of a node.
```

Comment:
Nit: It may be better to just say it returns _all_ children. 

Direct, thunk and special children are slightly weird, unexported concepts.